misc/vm_configs/xmls/config-xmls/apl-mrb/hybrid.xml

```diff
@@ -29,7 +29,7 @@
         <MCE_ON_PSC_DISABLED desc="Force to disable software workaround for Machine Check Error on Page Size Change.">n</MCE_ON_PSC_DISABLED>
         <IVSHMEM desc="IVSHMEM configuration">
             <IVSHMEM_ENABLED desc="Enable Share Memory between VMs by IVSHMEM.">n</IVSHMEM_ENABLED>
-            <IVSHMEM_REGION desc="the name, size, colon separated IDs of communication VMs of share memory region, separated by comma like this: hv:/shm_region_0, 0x200000, 0:2"></IVSHMEM_REGION>
+            <IVSHMEM_REGION desc="the name, size, IDs of communication VMs of share memory region, input format: hv:/&lt;shm name&gt;,&lt;shm size in MB&gt;,&lt;vm id:vm id[:vm id:...]&gt;"></IVSHMEM_REGION>
```

Comment:
As you've shown with this patch, we should not be putting comments in the XML files like this that need to be updated in 100 XML files when there's a change. We really should be documenting these XML parameters separately from the XML files themselves. (But that will likely need to be addressed in a future release.)

For now, you shouldn't be using entity codes such as ``&lt;`` in these comments since they're very hard to read and aren't really needed.  Instead,  you can do something simpler like this:
```
<IVSHMEM_REGION desc="the shared memory region name (starting with hv:/, size (in MB), and IDs of all VMs (separated by a colon) sharing this region, for example hv:/sharename,4,0:1:2</IVSHMEM_REGION>
```

Reply:
> As you've shown with this patch, we should not be putting comments in the XML files like this that need to be updated in 100 XML files when there's a change. We really should be documenting these XML parameters separately from the XML files themselves. (But that will likely need to be addressed in a future release.)
Agree with you. The description should be separated from XML files. @NanlinXie  has planned one task to implement it in the future release. 
> 
> For now, you shouldn't be using entity codes such as `&lt;` in these comments since they're very hard to read and aren't really needed. Instead, you can do something simpler like this:
> 
> ```
> <IVSHMEM_REGION desc="the shared memory region name (starting with hv:/, size (in MB), and IDs of all VMs (separated by a colon) sharing this region, for example hv:/sharename,4,0:1:2</IVSHMEM_REGION>
> ```
the entity codes are escape characters for xml. I will update this PR to update the desc following your comments.